stellar/operations.py

```diff
@@ -36,14 +36,9 @@ def copy_database(raw_conn, from_database, to_database):
     terminate_database_connections(raw_conn, from_database)
 
     if raw_conn.engine.dialect.name == 'postgresql':
-        raw_conn.execute(
-            '''
-                CREATE DATABASE "%s" WITH TEMPLATE "%s";
-            ''' %
-            (
-                to_database,
-                from_database
-            )
+        sqlalchemy_utils.functions.create_database(
+            '%s%s' % (raw_conn.engine.url, to_database),
+            template=from_database
```

Comment:
Because it's open! I had a quick look and didn't really understood what is it about, you do rely on sqlalchemy-utils. 

What's interesting is, why was I looking at the bug tracker. The reason is, I was wondering if I should add a command-line option to use another configuration file (different filename). The two or three times I thought this would be useful I ended up doing some other solution instead. 

This whole stellar package is interesting. The functionality is so small and provided by PostgreSQL already it makes me wonder if I really need this package. On the other hand there is something really tempting about just typing "stellar restore" and having things done. On another, stellar is the thing I learned "CREATE TABLE FROM TEMPLATE" about. 

Anyways.